manifests/globals.pp

```diff
@@ -106,7 +106,11 @@
     '93'    => '2.1',
     default => undef,
   }
-  $globals_postgis_version = pick($postgis_version, $default_postgis_version)
+  if $postgis_version {
```

Comment:
What is the reason for the rewrite of this?  This is the exact what pick is for.